latch_cli/services/local_dev.py

```diff
@@ -73,32 +53,39 @@ def _get_latest_image(pkg_root: Path) -> str:
     return f"{config.dkr_repo}/{ws_id}_{pkg_root.name}:{latest_version}"
 
 
-async def _copy_files(
-    ssh_conn: asyncssh.SSHClientConnection,
+async def _poll_rsync(
     pkg_root: Path,
+    ip: str,
+    ssh_port: str,
+    key_path: Path,
+    interval: float = 0.5,
 ):
-    # TODO(ayush) rsync
-
-    coroutines = []
-    for path in [
-        pkg_root.joinpath("wf"),
-        pkg_root.joinpath("data"),
-        pkg_root.joinpath("scripts"),
-    ]:
-        if path.exists():
-            coroutines.append(
-                asyncssh.scp(
-                    path,
-                    (ssh_conn, f"~/{pkg_root.name}"),
-                    recurse=True,
-                    block_size=16 * 1024,
-                )
-            )
-        else:
-            await aioconsole.aprint(f"Could not find {path} - skipping")
+    command = [
+        "rsync",
+        "-e",
+        f"'ssh -p {ssh_port} -i {str(key_path)} -o StrictHostKeyChecking=no'",
+        "-az",
+        "--delete",
+        f"{str(pkg_root)}",
+        f"root@{ip}:~/{pkg_root.name}/",
+    ]
+    total_fails = 0
+    while True:
+        outs = await asyncio.subprocess.create_subprocess_shell(
```

Comment:
```suggestion
        outs = await asyncio.subprocess.create_subprocess(
```
no need for shell here